Public/Src/Engine/Dll/DirectoryScrubber.cs

```diff
@@ -111,18 +75,61 @@ public static IEnumerable<string> CollapsePaths(IEnumerable<string> paths)
         /// <remarks>
         /// A directory path is valid to scrub if it is under a scrubbable mount.
         /// </remarks>
-        private bool ValidateDirectory(string directory, out SemanticPathInfo foundSemanticPathInfo)
+        private bool ValidateDirectory(MountPathExpander mpe, string directory, out SemanticPathInfo foundSemanticPathInfo)

```

Comment:
>mpe [](start = 57, length = 3)

nit: Given other argument is in understandable name, I think, instead of `mpe`, you should name it `expander` or `mountPathExpander`.

Reply:
changed